opencti-platform/opencti-graphql/src/modules/entitySetting/entitySetting-resolvers.ts

```diff
@@ -0,0 +1,20 @@
+import { entitySettingEditField, entitySettingsEditField, findAll, findById, findByType } from './entitySetting-domain';
+import type { Resolvers } from '../../generated/graphql';
+
+const entitySettingResolvers: Resolvers = {
+  Query: {
+    entitySetting: (_, { id }, context) => findById(context, context.user, id),
+    entitySettings: (_, args, context) => findAll(context, context.user, args),
+    entitySettingByType: (_, { targetType }, context) => findByType(context, context.user, targetType),
+  },
+  Mutation: {
+    entitySettingFieldPatch: (_, { id, input }, context) => {
+      return entitySettingEditField(context, context.user, id, input);
+    },
+    entitySettingsFieldPatch: (_, { ids, input }, context) => {
```

Comment:
Not sure why we need two mutation to do quite the same thing ? Single update = Array update with only one id ?

Reply:
You are perfectly right !
I will changed it